__tests__/index.spec.js

```diff
@@ -1,56 +1,136 @@
 const pluginTester = require('babel-plugin-tester').default;
```

Comment:
Can we rename this file to `index-test.js`. That seems to be a more conventional name.